docs/src/test/scala/docs/http/scaladsl/server/RejectionHandlerExamplesSpec.scala

```diff
@@ -68,6 +72,40 @@ class RejectionHandlerExamplesSpec extends RoutingSpec {
       }
     //#example-1
   }
+  
+  "example-2-all-exceptions-json" in {
+    //#example-json
+    import akka.http.scaladsl.model._
+    import akka.http.scaladsl.server.RejectionHandler
+
+    implicit def myRejectionHandler =
+      RejectionHandler.default
+        .mapRejectionResponse {
+          case res if res.entity.isInstanceOf[HttpEntity.Strict] =>
```

Comment:
Oh true. Changing (was a deep match here and got ugly fast, seems i didn't properly refactor)